src/mixins/extensions/LoadFieldComponents.js

```diff
@@ -43,7 +52,22 @@ export default {
         if (componentName === "FormImage") {
           this.registerVariable(element.config.variableName, element);
           delete properties.image;
-          properties[":image"] = this.byRef(element.config.image);
+
+          if (
+            !this.$store.getters["blobImagesModule/allBlobImages"][
+              element.config.name
```

Comment:
The image name (`element.config.name`) is not required for the image controls, also it is not unique; because of these it is not safe for indexing the image. Instead, consider:
- blobImageModule could store the image and return a unique incremental id
- generate a uuid
- use date time + random number
- md5(image)